docs/content/latest/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -0,0 +1,183 @@
+
+1. Enter this platform’s IP address or hostname (including the HTTP or HTTPS protocol prefix).
+
+    <br/>
+
+    ![Add an instance](/images/yp/high-availability/add-standby-instance.png)
+
+1. Copy the shared authentication key you generated on the active platform, and paste it into the Shared authentication key field. Double-check to make sure the keys match.
+
+    <br/>
+
+    ![Shared key field](/images/yp/high-availability/shared-key-field.png)
+
+1. On the active platform, click Add Instance, enter the new standby's IP address or hostname (including the HTTP or HTTPS protocol prefix), and click Continue.
+
+    <br/>
+
+    ![Add an instance](/images/yp/high-availability/add-standby-instance.png)
+
+1. On the new standby platform, wait for a replication interval (you set this when you [set up the active instance](#set-up-the-active-platform)), then refresh the page. The other instances in the HA cluster should now appear in the list of instances.
+
+    <br/>
+
+    ![Instance list](/images/yp/high-availability/instance-configuration.png)
+
+    If you click on other tabs, notice that they are greyed out. You can only use those tabs on the active instance.
+
+    <br/>
+
+    ![Tabs are greyed out](/images/yp/high-availability/standby-tabs-unavailable.png)
+
+### Promote a Standby Platform to Active
+
+1. On the standby platform you wish to promote to active, click the Make Active button in the upper-right corner of the "Replication Configuration" tab.
+
+    <br/>
+
+    ![Make active](/images/yp/high-availability/standby-make-active.png)
+
+1. Select the backup you want to restore (we strongly recommend you choose the most recent backup in the vast majority of cases!), and confirm that you want to promote this instance to active.
+
+    <br/>
+
+    The restore should take only a few seconds, and you'll be logged out when it's finished.
+
+1. To log back in, use the user credentials that you had configured on the previously active platform.
+
+    <br/>
+
+    Once you're logged in, you should see that all of the data has been restored into the platform including universes, users, metrics, alerts, task history, cloud providers, and so on.
+
+### Remove a Standby Platform
+
+To remove a standby platform from a high-availability cluster, you remove it from the active platform's list, and then delete the configuration from the platform to be removed.
+
+1. On the standby platform you wish to remove from the high-availability cluster, click the Delete Configuration button on the Admin tab, and confirm the deletion.
+
+    <br/>
+
+    ![Delete configuration](/images/yp/high-availability/delete-configuration.png)
+
+1. On the active platform's instance list, click the Delete Instance button for the standby instance to be removed, and confirm the deletion.
+
+    <br/>
+
+    ![Instance list](/images/yp/high-availability/instance-configuration.png)
+
+The standby platform is now a standalone instance again.
```

Comment:
I think we need to add a disclaimer that the info on this standby platform will most likely be out of date and we strongly recommend wiping out the state here (i.e. drop the yugaware DB in postgres + re-create it) before using this previously standby platform in standalone mode again.

Otherwise customers may run into some unforeseen issues with stale universe data/state.